docs/DeveloperGuide.md

```diff
@@ -8,8 +8,189 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```

Comment:
Hello! There isn't an introduction on what your application is about :/
As well as any acknowledgement is missing as well.